tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+bok_choy==0.3.2
```

Comment:
@e-kolpakov This requirement is likely being added because XBlock now requires it.

...Which reminds me that we're probably going to have to refactor all the XBlock tests to use that, now that it's becoming best practice, but we need not do that now.

Of course, that begs the question of if the right XBlock has been used for testing-- I know @bradenmacdonald is currently working on the version of this XBlock that has real children, but we know he's going to use an updated XBlock. We probably don't want that for this specific PR. Either way, the bok_choy requirement is probably better left deferred to XBlock to require. If it doesn't currently, it probably needs a fix, though I suspect the real issue is that @mtyaka may have pulled the latest version of XBlock without re-running its installer to install prereqs. I may be wrong, though.